addons/common/functions/fnc_claim.sqf

```diff
@@ -29,19 +29,10 @@ if (!isNull _owner && {!isNull _unit} && {_unit != _owner}) then {
 _target setVariable [QGVAR(owner), _unit, true];
 
 // lock target object
-if (_lockTarget) then {
-    private _canBeDisassembled = ([] isNotEqualTo getArray (configOf _target >> "assembleInfo" >> "dissasembleTo")) && { !([false, true] select (_target getVariable [QEGVAR(csw,assemblyMode), 0])) };
-    if (!isNull _unit) then {
-        [QGVAR(lockVehicle), _target, _target] call CBA_fnc_targetEvent;
-        if (_canBeDisassembled) then {
-            _target enableWeaponDisassembly false;
-        };
-    } else {
-        [QGVAR(unlockVehicle), _target, _target] call CBA_fnc_targetEvent;
-        if (_canBeDisassembled) then {
-            _target enableWeaponDisassembly true;
-        };
-    };
+if (_lockTarget && {!isNull _unit}) then {
+    [QGVAR(lockVehicle), _target, _target] call CBA_fnc_targetEvent;
+} else {
+    [QGVAR(unlockVehicle), _target, _target] call CBA_fnc_targetEvent;
```

Comment:
```suggestion
if (_lockTarget) then {
    if (!isNull _unit) then {
        [QGVAR(lockVehicle), _target, _target] call CBA_fnc_targetEvent;
    } else {
        [QGVAR(unlockVehicle), _target, _target] call CBA_fnc_targetEvent;
    };
```

I think it needs to be like this to keep old behaviour
or we'll call unlockVehicle everytime